### PR TITLE
HPCC-14841 Return more WU graph information in WUQueryDetails

### PR DESCRIPTION
**Pull request by wangkx**
The existing WsWorkunits.WUQueryDetails only returns graph ID.
The WsWorkunits.WUInfo returns gragh type, label, and timing.
In this fix, the functions of retrieving graph information in
WsWorkunits.WUInfo are collected into 2 methods which are used
in WUQueryDetails also.

Signed-off-by: wangkx kevin.wang@lexisnexis.com

### CODE REVIEW

**Comment by wangkx** [comment]
@afishbeck please review.

**Comment by wangkx** [comment]
This PR may go to 6.4.

**Review by afishbeck — APPROVED** [review]
@wangkx Looks fine.

**Comment by wangkx** [comment]
@richardkchapman please merge this PR to 6.4.

File: esp/scm/ws_workunits.ecm [hunk 1]
```diff
@@ -1471,9 +1471,11 @@ ESPresponse [exceptions_inline] WUQueryDetailsResponse
     [min_ver("1.46")] string CompileTime;
     [min_ver("1.46")] ESParray<string> LibrariesUsed;
     [min_ver("1.46")] int CountGraphs;
-    [min_ver("1.46")] ESParray<string> GraphIds;
+    [min_ver("1.46"), depr_ver("1.62")] ESParray<string> GraphIds;
```
Comment (richardkchapman) [inline esp/scm/ws_workunits.ecm]: Does EclWatch code need to change (does it use default version of these interfaces?)

@GordonSmith

Comment (GordonSmith) [inline esp/scm/ws_workunits.ecm]: ECLWatch does use this response item and does use the default version (no version passed in) - I think that means its ok?

**Comment by afishbeck** [comment]
If it still works as designed an HTTP-GET without version will reflect all
the most recent changes.  A SOAP request on the other hand will stay at the
oldest version until one is specified.  Some SOAP clients are very
sensitive to even new tags being added.

If short on time, the quickest fix would be for EclWatch to start sending
in the version number it was coded against.  Then it could be modified to
reflect these changes when appropriate.

File: esp/scm/ws_workunits.ecm (hunk 1 above)
Comment (wangkx) [inline esp/scm/ws_workunits.ecm]: EclWatch does need to recode to use the new ones which give more graph information as in WsWorkunits.WUInfo. The https://track.hpccsystems.com/browse/HPCC-14837 is for recoding EclWatch.

**Comment by richardkchapman** [comment]
Also needs rebasing for a clean merge

**Comment by wangkx** [comment]
@richardkchapman this PR is re-based and re-tested.

**Comment by richardkchapman** [comment]
@GordonSmith Any of this going to cause you any issues?

**Comment by GordonSmith** [comment]
Only in a good way - I will open a Jira for ECL Watch

**Pull request merged**